apisix/plugins/ext-plugin/init.lua

```diff
@@ -152,6 +165,80 @@ end
 _M.receive = receive
 
 
+local generate_id
+do
+    local count = 0
+    local MAX_COUNT = lshift(1, 22)
+
+    function generate_id()
+        local wid = worker_id()
+        local id = lshift(wid, 22) + count
+        count = count + 1
+        if count == MAX_COUNT then
+            count = 0
```

Comment:
Does the id rotate will influence the correctness?

Reply:
The id here is only used for logging purpose. In the given time window, it is rare to have two repeated id so we can distinguish them.

Reply:
OK.